src/System.Runtime/tests/System/Lazy.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Runtime.Tests
 {
-    public static class LazyInitializerTests
```

Comment:
Note that this class actually does have some test cases for the "LazyInitializer" class, which is technically in the System.Threading contract. We should probably move those over into the other test project when it's merged in, because it doesn't look like it technically has anything to do with Lazy itself.